src/Tests/Microsoft.NET.ToolPack.Tests/DefaultPackAndResultNugetPackageNuGetPackageFixture.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using NuGet.Packaging;
+using System.Xml.Linq;
+using System.Runtime.CompilerServices;
+using System;
+using NuGet.Frameworks;
+
+namespace Microsoft.NET.ToolPack.Tests
+{
+    public class DefaultPackWithShimsAndResultNugetPackageNuGetPackagexFixture : IDisposable
+    {
+        public const string _customToolCommandName = "customToolCommandName";
+        private const string _packageVersion = "1.0.0";
+        public DefaultPackWithShimsAndResultNugetPackageNuGetPackagexFixture()
+        {
+        }
+
+        public void Dispose()
+        {
+        }
+
+        public ITestOutputHelper Log { get; private set; }
+        public TestAssetsManager TestAssetsManager { get; private set; }
+        public void Init(ITestOutputHelper log, TestAssetsManager testAssetsManager)
+        {
+            Log = log;
+            TestAssetsManager = testAssetsManager;
+        }
+        public Dictionary<(bool multiTarget, string targetFramework), string> assetMap
+            = new Dictionary<(bool multiTarget, string targetFramework), string>();
+
+        public string GetAsset(bool multiTarget, string targetFramework)
```

Comment:
I'd rename this method to make it clear that it is returning the path to a .nupkg, perhaps `GetTestToolPackagePath`.

Reply:
Renamed to suggested